components/FileTable.tsx

```diff
@@ -103,15 +107,14 @@ export default class FileTable extends React.Component<IFileTableProps> {
             {
                 name: 'Biospecimen',
                 selector: (file: Entity) => {
-                    return file.primaryParents
-                        ? _.uniq(
-                              file.primaryParents.map(
-                                  (_file) =>
-                                      _file.biospecimen!.HTANBiospecimenID
-                              )
+                    const biospecimens = file.primaryParents
```

Comment:
we shouldn't optimize as part of this PR but i think we should do this computation once up front for all files

Reply:
good point